setup.cfg

```diff
@@ -1,2 +1,60 @@
 [metadata]
+author = Argo AI
+author_email = argoverse-api@argo.ai
+classifiers = 
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: POSIX
+    Operating System :: MacOS
+    Programming Language :: Python :: 3
+    Topic :: Scientific/Engineering :: Artificial Intelligence
 description_file = README.md
+keywords =
+    self-driving-car
+    dataset-tools
+license = MIT
+name = argoverse
+url = https://www.argoverse.org
+version = attr: argoverse.VERSION
+
+[options]
+zip_safe = False
```

Comment:
why do we want zip_safe = False?

Reply:
This was just a default option listed here: https://setuptools.readthedocs.io/en/latest/userguide/declarative_config.html?highlight=zip_safe. I don't have a strong opinion on it. `setuptools` will automatically try to determine the setting if we remove it, which might be useful.